website/docs/d/healthcare_dicom.html.markdown

```diff
@@ -51,6 +63,29 @@ An `authentication` supports the following:
 
 * `audience` - The intended audience to receive authentication tokens for the service. The default value is <https://dicom.azurehealthcareapis.azure.com>
 
+---
+
+A `cors` supports the following:
+
+* `allowed_origins` - A list of allowed origins for CORS.
+
+* `allowed_headers` - A list of allowed headers for CORS.
+
+* `allowed_methods` - A list of allowed methods for CORS.
+
+* `max_age_in_seconds` - The maximum age in seconds for the CORS configuration.
+
+* `allow_credentials` - Whether to allow credentials in CORS.
+
+---
+
+A `storage` block supports the following:
+
+* `file_system_name` - The filesystem name of connected storage account.
+
+* `storage_account_id` - The resource id of connected storage account.
```

Comment:
```suggestion
* `storage_account_id` - The resource ID of connected storage account.
```

Reply:
Updated the doc with required changes & other parts where it is id rather than ID